<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DiarioBase</class>
 <widget class="QWidget" name="DiarioBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Diario</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_diario.xpm</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="pixmapLabel1" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../bulmages-recursos-graficos.qrc" >:/BulmaCont32x32/images/pendientes/i_diario.xpm</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_guardar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Guardar albaran</string>
          </property>
          <property name="whatsThis" >
           <string>Guardar albaran</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Borrar albaran</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar albaran</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_filtrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Imprimir albaran</string>
          </property>
          <property name="whatsThis" >
           <string>Imprimir albaran</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_find.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>154</width>
            <height>36</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="mui_verpedidocliente" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Ver pedido de cliente</string>
          </property>
          <property name="whatsThis" >
           <string>Ver pedido de cliente</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_facturar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Pasar a factura</string>
          </property>
          <property name="whatsThis" >
           <string>Pasar a factura</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_agregarafactura" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Agregar a factura</string>
          </property>
          <property name="whatsThis" >
           <string>Agregar a factura</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_cobrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Agregar a factura existente</string>
          </property>
          <property name="whatsThis" >
           <string>Agregar a factura existente</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel1_3" >
          <property name="text" >
           <string>Fecha inicial</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaFecha" name="m_fechainicial1" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel1_2_2" >
          <property name="text" >
           <string>Fecha final</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaFecha" name="m_fechafinal1" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>191</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="DiarioSubForm" name="mui_list" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>340</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel2" >
       <property name="text" >
        <string>Totales:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="totaldebe" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>197</red>
           <green>194</green>
           <blue>197</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>226</red>
           <green>224</green>
           <blue>226</blue>
          </color>
          <color>
           <red>98</red>
           <green>96</green>
           <blue>98</blue>
          </color>
          <color>
           <red>131</red>
           <green>129</green>
           <blue>131</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>127</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </active>
         <inactive>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>197</red>
           <green>194</green>
           <blue>197</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>226</red>
           <green>222</green>
           <blue>226</blue>
          </color>
          <color>
           <red>98</red>
           <green>96</green>
           <blue>98</blue>
          </color>
          <color>
           <red>131</red>
           <green>129</green>
           <blue>131</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>127</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>0</green>
           <blue>255</blue>
          </color>
          <color>
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </inactive>
         <disabled>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>197</red>
           <green>194</green>
           <blue>197</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>226</red>
           <green>222</green>
           <blue>226</blue>
          </color>
          <color>
           <red>98</red>
           <green>96</green>
           <blue>98</blue>
          </color>
          <color>
           <red>131</red>
           <green>129</green>
           <blue>131</blue>
          </color>
          <color>
           <red>128</red>
           <green>128</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>128</red>
           <green>128</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>127</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>0</green>
           <blue>255</blue>
          </color>
          <color>
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </disabled>
        </palette>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="totalhaber" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>197</red>
           <green>194</green>
           <blue>197</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>226</red>
           <green>224</green>
           <blue>226</blue>
          </color>
          <color>
           <red>98</red>
           <green>96</green>
           <blue>98</blue>
          </color>
          <color>
           <red>131</red>
           <green>129</green>
           <blue>131</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>127</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </active>
         <inactive>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>197</red>
           <green>194</green>
           <blue>197</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>226</red>
           <green>222</green>
           <blue>226</blue>
          </color>
          <color>
           <red>98</red>
           <green>96</green>
           <blue>98</blue>
          </color>
          <color>
           <red>131</red>
           <green>129</green>
           <blue>131</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>127</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>0</green>
           <blue>255</blue>
          </color>
          <color>
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </inactive>
         <disabled>
          <color>
           <red>128</red>
           <green>128</green>
           <blue>128</blue>
          </color>
          <color>
           <red>197</red>
           <green>194</green>
           <blue>197</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>226</red>
           <green>222</green>
           <blue>226</blue>
          </color>
          <color>
           <red>98</red>
           <green>96</green>
           <blue>98</blue>
          </color>
          <color>
           <red>131</red>
           <green>129</green>
           <blue>131</blue>
          </color>
          <color>
           <red>128</red>
           <green>128</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>128</red>
           <green>128</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>127</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>128</blue>
          </color>
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
          <color>
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
          <color>
           <red>255</red>
           <green>0</green>
           <blue>255</blue>
          </color>
          <color>
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </disabled>
        </palette>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="5" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>BusquedaFecha</class>
   <extends></extends>
   <header>../../../bulmalib/busquedafecha.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>DiarioSubForm</class>
   <extends>QFrame</extends>
   <header>diariosubform.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages-recursos-graficos.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_filtrar</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>155</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
